### PR TITLE
Improve contributing

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

**Comment by mwestphal** [comment]
@snoyer please review :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,8 +4,11 @@ F3D welcomes all contributors, regardless of skill level or experience!
...
 
-To contribute to F3D as a user, you can just use F3D and report any issues or feature ideas.
-Please open your own issue to share any issues or ideas you may have, and comment on any related issues.
+Contributing to F3D can be as simple as pointing out a spelling mistake on the website, 
+reporting a bug you encountered, or suggesting a new feature you feel would improve the application.
+
+We use [Github](https://github.com/f3d-app/f3d) to manage the project and you can report new issues or weigh 
+in on existing ones [over there](https://github.com/f3d-app/f3d/issues).
 
 Also, do not hesitate to join our [discord](https://discord.f3d.app) !
```
Comment (snoyer) [inline CONTRIBUTING.md:13]:
```suggestion
Also, do not hesitate to join our [discord](https://discord.f3d.app)!
```
No need to make it weird, this is not French ;p

**Review by snoyer — APPROVED** [review]
looks good

**Comment by mwestphal** [comment]
can you approve @snoyer ? you should be able to do that in the "Files Changed" tab.

**Review by snoyer — APPROVED** [review]

**Review by Meakk — APPROVED** [review]

**Pull request merged**